### ISSUE TITLE
Peut-on garder le compteur de news et l'agenda jour par jour ?

### ISSUE THREAD

**Issue by zynaps59** [Aqualizr/beta#44]
Salut, 
Voila, peut-on garder le compteur de news et l'agenda jour par jour comme sur le comptoir de l'info svp 
?

![capture](https://cloud.githubusercontent.com/assets/13525585/8926611/57554e50-3511-11e5-8ee4-24c95894cb4c.PNG)

Cordialement.

**Comment by Zebwest** [comment]
OUi, je serais assez d'accord sur le principe, même si il est peut être possible de rendre cela plus moderne...

**Comment by ysoudy** [comment]
on pourrait le faire dans la zone de droite effectivement, on va mettre çà dans la todo list ;)
